datalad/distributed/create_sibling_ghlike.py

```diff
@@ -507,6 +507,11 @@ def repo_create_response(self, r):
                 status='error',
                 message=('unauthorized: %s', response.get('message')),
             )
+        elif r.status_code == requests.codes.internal_server_error:
+            return dict(
+                status='error',
+                message=response.get('message', '').strip() or None,
```

Comment:
note that if server did not provide any message -- user end up without any clue what actually has happened, not even an error code (500), may be
```suggestion
                message=response.get('message', '').strip() or 'Server returned error code %d without any further information' % requests.codes.internal_server_error,
```
or alike?